Projet/physic/ContactHandler.h

```diff
@@ -14,12 +14,13 @@ class ContactHandler : public physx::PxSimulationEventCallback, public physx::Px
     virtual void onConstraintBreak(physx::PxConstraintInfo* constraints, physx::PxU32 count) {}
```

Comment:
MaJuScUlEs

Reply:
I not touch this, but I think we want to keep a same format as the onContact function, and they don't take uppercase.

Reply:
+1 we should keep physx convention when interfacing with it like here